tests/microos/journal_check.pm

```diff
@@ -16,43 +16,39 @@ use warnings;
 use testapi;
 use microos;
 use version_utils 'is_opensuse';
+use Mojo::JSON qw(decode_json);
 
 sub run {
     my $self = shift;
+    my $res  = Mojo::UserAgent->new->get(data_url('journal_check/bug_refs.json'))->result;
+    $res->is_success or die "Could not download file with bug refs";
+    my %bug_pattern;
+    my $product = get_var('DISTRI');
+    my $version = get_var('VERSION');
+
+    my $bugs = decode_json($res->body);
+    foreach my $bugid (keys %$bugs) {
+        $bugs->{$bugid}->{products}->{$product} or next;
+        if (ref $bugs->{$bugid}->{products}->{$product} eq ref []) {
+            foreach my $ver (@{$bugs->{$bugid}->{products}->{$product}}) {
+                $bug_pattern{$bugid} = $bugs->{$bugid}->{description} if ($ver eq $version);
+            }
+        }
+        else {
+            $bug_pattern{$bugid} = $bugs->{$bugid}->{description};
+        }
+    }
+    use Data::Dumper;
```

Comment:
shall we move this to where all the other `use` are?

Reply:
It will go away, part of debugging leftover :)